admin-tools.Dockerfile

```diff
@@ -13,7 +13,11 @@ COPY ./temporal/go.mod ./temporal/go.sum ./temporal/
 RUN (cd ./temporal && go mod download all)
 
 # build
-COPY . .
+COPY ./temporal ./temporal
+# Git info is needed for Go build to attach VCS information properly.
+# See the `buildvcs` Go flag: https://pkg.go.dev/cmd/go
+COPY ./.git ./.git
+COPY ./.gitmodules ./.gitmodules
```

Comment:
We can leave this for now.
I'm curious to know how we use that `buildvcs` information.

Reply:
The info is passed through to some internal variables that are used in places like CLI info:
https://github.com/temporalio/temporal/blob/af9130dcdbede1a8983069fde69d7c139d9d7b22/common/build/build.go#L42